README.adoc

```diff
@@ -51,23 +51,27 @@ The plugin provides bootstrap script that creates sample project with:
 * proper source file layout
 * sample https://gatling.io/docs/current/general/simulation_structure/[Simulation class], demonstrating sufficient `Gatling` functionality
 
-For this quickstart - `git` and `curl` must be instlled and available in `$PATH`.
+For this quickstart - `git` and `curl` must be installed and available in `$PATH`.
 
-.Generate sample Gradle project and run Gatling simulation
-[source, bash]
-----
-curl -sL https://git.io/Jf2Uk | bash -s ~/sample-gradle-gatling
-----
 
-.Navigate to new project folder and run all simulations
+[TIP]
+====
+Sample project skeleton should be adjusted after a new plugin will be released.
+====
+
```

Comment:
Should not be in git history

Reply:
1. the id of plugin is already agreed with @slandelle and it's already in source tree. It is `io.gatling.gradle`
1. I didn't understand the purpose of this comment, plz clarify.
> Should not be in git history

Reply:
1. Yes, `io.gatling.gradle` will be the id of the plugin.
    Right now, it is in "pending approval" status.
![2020-08-26-113807_844x225_scrot](https://user-images.githubusercontent.com/693554/91416436-ad8d4b00-e84f-11ea-91e4-d665a106ef5d.png)

2. No need to have a task ("should be ... after") in the source tree. We can avoid this diff in the git history. 
  We can have a task in [gatling/gatling issues](https://github.com/gatling/gatling/labels/Module%20-%20Gradle%20Plugin) (note: I added gatling/gatling#3940 )

Reply:
removed